src/routes/manage-selections/components/SelectionItem.tsx

```diff
@@ -41,28 +36,20 @@ const SelectionItem = ({ identifier, deleteItem }: PropsType): JSX.Element => {
     )
   }
 
-  if (!data) {
-    let errorMessage = getLabels().general.error.unknown
-    if (error?.message === ForbiddenError) {
-      errorMessage = getLabels().home.errorLoadCustomDiscipline
-    } else if (error?.message === NetworkError) {
-      errorMessage = `${getLabels().general.error.noWifi} (${error.message})`
-    }
-    return <ListItem title={errorMessage} />
+  if (!data && error?.message === NetworkError) {
+    return <ListItem title={`${getLabels().general.error.noWifi} (${error.message})`} hideRightChildren />
```

Comment:
(optional) We can also just pass empty tags here `<></>` for the prop rightChildren, so we can get rid of this additional hideRightChildren prop.

Reply:
Shouldn't rather just nothing be passed? I.e. the prop should be optional I think.

Reply:
@SteffiStuffel that was my first thought as well, either `<></>` or just `null`. However, eslint complains (`LUN-387-add-option-to-delete-profession-on-error`). 

If you want me to, I'll disable the error for that line, I simply thought this was the cleaner solution.

Reply:
better to disable the error than to add unneeded code imo :)

Reply:
Getting conflicting messages here 😄 

Is it really unnecessary code if the alternative is actively suppressing an error? Plus, `ListItem` is currently being used in a couple of different contexts, I can imagine there are other situations where we would not want to have an icon.

Though having none by default might make more sense.

Reply:
You are free to choose, which solution you prefer. Both is fine.